### PR TITLE
add footnotes to document event sources in the address books and logbooks

### PR DESCRIPTION
**Pull request by rlskoeser**
Data migration to add footnotes to all remaining events without footnotes, and create log entries to document the footnote creation. Events should be associated with address books based on tags in the notes, all remaining events without footnotes are associated with the logbooks. See #702 for details

### CODE REVIEW

File: mep/accounts/migrations/0034_add_event_footnotes.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
+# Generated by Django 2.2.11 on 2020-12-08 14:53
+
+from django.db import migrations
+from django.contrib.admin.models import ADDITION
+from django.conf import settings
+
+
+def add_missing_footnotes(apps, schema_editor):
+    '''
+    '''
```
Comment (thatbudakguy) [inline mep/accounts/migrations/0034_add_event_footnotes.py:10]: does this need a docstring?

File: mep/accounts/migrations/0034_add_event_footnotes.py [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+                object_id=footnote.pk,
+                object_repr='Footnote on event %s for %s' %
+                            (event.pk, addressbook_1936.bibliographic_note),
+                change_message='Address book footnote created based on 36ADD tag',
+                action_flag=ADDITION)
+
+    if logbooks:
+        # find all remaining events without footnotes — from the logbooks
+        logbooks_events = Event.objects.exclude(notes__contains='36ADD') \
+                               .exclude(pk__in=events_with_footnotes)
```
Comment (thatbudakguy) [inline mep/accounts/migrations/0034_add_event_footnotes.py:87]: do you still need the first `exclude()` here?

**Review by thatbudakguy — APPROVED** [review]
looks good; the actual migration seems pretty straightforward.

**Review by kmcelwee — COMMENTED** [review]
Does this mean that we've inadvertently added a feature where if a user adds the tag "ADD36" to notes, the footnote will be appended during the next migration?

**Pull request merged**